transformer_engine/__init__.py

```diff
@@ -2,6 +2,19 @@
 #
 # See LICENSE for license information.
 
+# pylint: disable=missing-module-docstring
+# pylint: disable=missing-function-docstring
+# pylint: disable=missing-class-docstring
```

Comment:
Why are these needed? Current version of __init__ passes without these disables and I don't see any change here that would change that

Reply:
It is caused by misaligned pylint setting between `main` and internal devel branch.
Now, it should be fixed by 5615c9b